src/app/programs/page.tsx

```diff
@@ -0,0 +1,30 @@
+import ProgramCard from "./programCard";
+
+export default async function ProgramsPage() {
+  interface Program {
+    name: string;
+    color: string;
+    image: string;
+    description: string;
+  }
+
+  const programRequest = await fetch("http://localhost:3000/api/programs");
```

Comment:
This being hardcoded will give an error when someone is not running the app on localhost:3000.

Use this solution from the job branch that uses a [middleware.ts](https://github.com/INIT-Build-Web-Dev-3-Spring-2024/init-website/blob/job/src/middleware.ts) file and a [dynamic fetch url](https://github.com/INIT-Build-Web-Dev-3-Spring-2024/init-website/blob/c60f88c375cfbef65849c73900df69094882fb3d/src/app/jobs/%40companies/page.tsx#L12)